spec/util/vmdb-logger_spec.rb

```diff
@@ -95,7 +95,7 @@
   end
 
   context "long messages" do
-    let(:logger) { VMDBLogger.new(@log) }
+    let(:logger) { VMDBLogger.new(File.expand_path(File.join(File.dirname(__FILE__), 'data/long_messages.log'))) }
```

Comment:
This is odd. Can we test an instance of the formatter directly since that's all this one spec seems to be doing? Then we can drop this `let` altogether, right?

Reply:
I think line 102 would become:
```ruby
 _, message = VMDBLogger::Formatter.new.call(:error, Time.now.utc, "", msg).split("-- : ")
```

Reply:
Done!